regression/benchmark/benchmarks.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
```

Comment:
Rename this file to `run_all.sh`, which is more self-describing than `benchmarks.sh`.